### PR TITLE
Upgraded to Vulkano 0.11. Added DejaVuSans font to repo

### PR DESCRIPTION
**Pull request by jsarasin**
- Changed from vulkano-shaders-derive to vulkano-shaders.
- Updated shader definitions in the library and example
- Included the DejaVuSans.ttf as well as the license
- Bit of a hack to get the example to compile.

### CODE REVIEW

**Comment by andrewhickman** [comment]
Thanks for the PR!

The vulkano version update looks good. I'd rather not add the font file though, just to avoid any issues with the license.

File: src/cache.rs [hunk 1]
```diff
@@ -87,7 +87,8 @@ impl<'font> GpuCache<'font> {
             };
 
             *result = upload(rect, data, queue, cmd, img, buf).map(Some);
-        })
+        });
```
Comment (andrewhickman) [inline src/cache.rs]: Should propagate the error with
```suggestion
        })?;
```

**Review by andrewhickman — CHANGES_REQUESTED** [review]

**Comment by jsarasin** [comment]
Changes made, font removed. Not sure if I need to do anything else on my end.

**Comment by andrewhickman** [comment]
Thats great thanks, I will publish a new package this evening

**Pull request merged**